src/components/common/Layout/Layout.jsx

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
 import Header from '../Header/Header';
+import useAuth from 'hooks/useAuth';
+import Map from 'components/common/Map/Map';
+import PhoneAppSection from 'components/common/PhoneAppSection/PhoneAppSection';
```

Comment:
Maybe separate local imports from the ones from libraries, and place relative paths at the bottom

Reply:
Excelent correction